Failed to build on Raspberry Pi

Hello,

Compilation of baresip v2.3.0 is failing on Raspberry Pi
Here is the result of make bareinfo
```
info - baresip version 2.3.0
  MODULES:       account auconv auresamp contact ctrl_tcp debug_cmd ebuacip echo fakevideo httpd ice menu mwi natpmp presence rtcpsummary selfview serreg srtp stun turn uuid vidbridge vidinfo vumeter mixausrc mixminus multicast aubridge aufile ausine cons dtls_srtp evdev g711 httpreq l16 netroam snapshot stdio syslog v4l2 omx
  MACHINE:       arm-linux-gnueabihf
  ARCH:          arm6
  OS:            linux
  BUILD:         build-arm6
  PKG_CONFIG:    0.29.2
  CCACHE:
  CC:            gcc 10 (10.x)
  CFLAGS:        -Wall -Wextra -Wmissing-declarations -Wmissing-prototypes -Wstrict-prototypes -Wbad-function-cast -Wsign-compare -Wnested-externs -Wshadow -Waggregate-return -Wcast-align -Wold-style-definition -Wdeclaration-after-statement -Wvla  -g -fPIC -DLINUX -DOS="linux" -std=c11 -DHAVE_ATOMIC -pedantic -DARCH="arm6" -DUSE_OPENSSL -DUSE_TLS -DUSE_OPENSSL_DTLS -DUSE_DTLS -DUSE_OPENSSL_SRTP -DUSE_DTLS_SRTP -DUSE_ZLIB -DHAVE_PTHREAD -DHAVE_GETIFADDRS -DHAVE_STRERROR_R -DHAVE_GETOPT -DHAVE_NET_ROUTE_H -DHAVE_SYS_SYSCTL_H -DHAVE_STDBOOL_H -DHAVE_INET6 -DHAVE_SYSLOG -DHAVE_RESOLV -DHAVE_EXECINFO -DHAVE_FORK -DHAVE_PWD_H -DHAVE_POLL      -DHAVE_SELECT -DHAVE_SELECT_H -DHAVE_SETRLIMIT -DHAVE_SIGNAL -DHAVE_SYS_TIME_H -DHAVE_EPOLL -DHAVE_UNAME -DHAVE_UNISTD_H -DHAVE_STRINGS_H -DVERSION="2.3.0" -DVER_MAJOR= -DVER_MINOR= -DVER_PATCH=  -I. -Iinclude -I/usr/local/include/re -I/usr/local/include/rem -DPREFIX="/usr/local" -DMOD_PATH="/usr/local/lib/baresip/modules" -DSHARE_PATH="/usr/local/share/baresip"
  DFLAGS:        -MD -MF info -MT info
  LFLAGS:        -fPIC
  SH_LFLAGS:     -shared
  MOD_LFLAGS:    -L/usr/local/lib
  APP_LFLAGS:    -rdynamic
  LIBS:          -ldl -lssl -lcrypto -lz -lpthread -L/usr/local/lib -L/usr/local/lib -lrem -lm
  LIBRE_MK:      ../re-2.3.0/mk/re.mk
  LIBRE_PATH:
  LIBRE_INC:     /usr/local/include/re
  LIBRE_SO:      /usr/local/lib
  OPENSSL_OPT:
  USE_OPENSSL:   yes
  USE_OPENSSL_AES:   yes
  USE_OPENSSL_HMAC:  yes
  USE_TLS:       yes
  USE_DTLS:      yes
  USE_DTLS_SRTP: yes
  USE_ZLIB:      yes
  GCOV:
  GPROF:
  CROSS_COMPILE:
  SYSROOT:       /usr
  SYSROOT_ALT:
  LIB_SUFFIX:    .so
  MOD_SUFFIX:    .so
  BIN_SUFFIX:
  LIBREM_PATH:   /usr/local
  LIBREM_INC:    /usr/local/include/rem
  LIBREM_SO:     /usr/local/lib
```
The error I encountered is the following:
```
  CC [M]  build-arm6/modules/omx/omx.o
In file included from modules/omx/omx.c:10:
modules/omx/omx.h:9:10: fatal error: IL/OMX_Core.h: No such file or directory
    9 | #include <IL/OMX_Core.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
make: *** [mk/mod.mk:49: build-arm6/modules/omx/omx.o] Error 1
```
Operating system information:
```
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```
I think this is Raspberry Pi Bullseys specific issue.
In `mk/modules.mk`, there is a line
```
USE_OMX_RPI  := $(shell $(call CC_TEST,bcm_host.h))
```
enabling omx module, but I couldn't find `/usr/include/IL/OMX_Core.h` file (actually, there is no `/usr/include/IL` directory) on my Pi.

I also tried installing the development package for OMX (libomxil-bellagio-dev). It placed OMX_Core.h file under `/usr/include`, but not under `/usr/include/IL`, so the same compilation error happened.

koki-takasaki writes:

> The error I encountered is the following:
> ```
>   CC [M]  build-arm6/modules/omx/omx.o
> In file included from modules/omx/omx.c:10:
> modules/omx/omx.h:9:10: fatal error: IL/OMX_Core.h: No such file or directory
>     9 | #include <IL/OMX_Core.h>
>       |          ^~~~~~~~~~~~~~~
> compilation terminated.
> make: *** [mk/mod.mk:49: build-arm6/modules/omx/omx.o] Error 1
> ```

I don't know why IL is in the path.  In Debian 11, package

libomxil-bellagio-dev

includes /usr/include/OMX_Core.h

If you have it installed, try without IL in the path.

--- a/mk/modules.mk
+++ b/mk/modules.mk
@@ -124,7 +124,7 @@ endif
 USE_X11      := $(shell $(call CC_TEST_AND,X11/Xlib.h,X11/extensions/XShm.h))
 USE_ZRTP     := $(shell $(call CC_TEST,libzrtp/zrtp.h))
 USE_VPX      := $(shell $(call CC_TEST,vpx/vp8.h))
-USE_OMX_RPI  := $(shell $(call CC_TEST,bcm_host.h))
+USE_OMX_RPI  := $(shell $(call CC_TEST_AND,bcm_host.h,IL/OMX_Core.h))
 
 USE_OMX_BELLAGIO := $(shell $(call CC_TEST,OMX_Core.h))
 USE_WEBRTC_AEC   := $(shell \

